src/test/java/org/jpeek/calculus/java/Lcom4Test.java

```diff
@@ -65,18 +64,19 @@ public void createsXmlCalculusWithXpaths() throws IOException {
         ).affirm();
     }
 
-    @Test
+    @ParameterizedTest
     @Disabled
-    public void methodMethodCalls() throws Exception {
+    @CsvFileSource(resources = "/org/jpeek/calculus/java/lcom4-params.csv")
+    public void methodMethodCalls(final String file, final String value) throws Exception {
         final XML result = new Lcom4().node(
```

Comment:
@HDouss I think `Lcom4` should have a constructor or several constructors to accept parameters and  the only its interfaced method `node` should be parameterless.
I mean, `Calculus ` should have:
```
public interface Calculus {
   
    XML node()
        throws IOException;

}
```

and `Lcom4` should have a cosntructor to accept the values.